### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by hipexscape** [ROM-builders/temporary#23395] [General]
I want to ask how can I upload these roms to my sf

**Comment by Apon77** [comment]
Download and upload. Simple

**Reply by hipexscape** [reply]
Yea i can do that but i have limited internet for now 
can we modify build.sh so that it can upload roms to my sf ?
@Apon77

**Reply by Apon77** [reply]
use google cloud shell/github action to do so, they have unlimited internet.
